src/client/componentsMealsharing/AllReservations.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import "../App.css";
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+function AllReservations() {
+  const [fetchData, setFetchData] = useState([]);
+
+  useEffect(() => {
+    fetchDataResult();
+  }, []);
+
+  const fetchDataResult = async () => {
+    const response = await fetch("http://localhost:3000/api/reservations");
+    const Data = await response.json();
+    setFetchData(Data);
+  };
+
+  //it seems like not working????
+  const onDeleteBookInfo = (e) => {
+    fetch(`http://localhost:3000/api/meals/${e}`, {
```

Comment:
I would rename the variable to be a bit more explanatory 
```suggestion
  const onDeleteBookInfo = (bookingId) => {
    fetch(`http://localhost:3000/api/meals/${bookingId}`, {
```